packages/shared/providers.yaml

```diff
@@ -4893,6 +4893,65 @@ sage:
         base_url: https://api.accounting.sage.com
     docs: https://docs.nango.dev/integrations/all/sage
 
+sage-intacct:
+    display_name: Sage Intacct
+    categories:
+        - accounting
+        - erp
+    auth_mode: TWO_STEP
+    proxy:
+        base_url: https://api.intacct.com/ia/xml/xmlgw.phtml
+    token_url: https://api.intacct.com/ia/xml/xmlgw.phtml
+    body_format: xml
+    token_params:
+        request:
+            control:
+                senderid: ${credential.senderId}
+                password: ${credential.senderPassword}
+                controlid: ${timestamp}
+                uniqueid: false
+                dtdversion: '3.0'
+                includewhitespace: false
+            operation:
+                authentication:
+                    login:
+                        userid: ${credential.userId}
+                        companyid: ${credential.companyId}
+                        password: ${credential.userPassword}
+                content:
+                    function:
+                        $:
+                            controlid: '{{$guid}}'
+                        getAPISession: ''
+    token_headers:
+        Content-Type: application/xml
+    token_response:
+        token: response.operation.result.data.api.sessionid
+        token_expiration: response.operation.authentication.sessiontimeout
+        token_expiration_strategy: expireAt
+    docs: https://docs.nango.dev/integrations/all/sage
+    credentials:
+        senderId:
+            type: string
+            title: Sender ID
+            description: Your Sage Intacct Sender ID
+        senderPassword:
+            type: string
+            title: Sender Password
+            description: Your Sage Intacct Sender Password
+        userId:
+            type: string
+            title: User ID
+            description: Your Sage Intacct User ID
+        companyId:
+            type: string
+            title: Company ID
+            description: Your Sage Intacct Company ID
+        userPassword:
```

Comment:
what's the diff between userPassword and senderPassword?

Reply:
The sender password is for the web service developer license, which has a combination of the sender Id and sender password, while the userPassword is part of the login credentials for the company to which you will be sending API calls. 
https://developer.intacct.com/web-services/your-first-api-calls/#prerequisites

Reply:
Alright, this will be clearer with the doc later